src/main/java/codeu/model/data/User.java

```diff
@@ -58,4 +58,9 @@ public String getPasswordHash() {
   public Instant getCreationTime() {
     return creation;
   }
+
+  @Override
+  public int compareTo(User otherUser) {
+    return getCreationTime().compareTo(otherUser.getCreationTime());
```

Comment:
This won't result in a unique ordering if 2 users happen to be created with the exact same creation time.  I believe you need to have something along the lines of a series of  "if (var.equals(otherUser.var) { return var.compareTo(otherUser.var); } " for each member variable in the class. Based on https://docs.oracle.com/javase/8/docs/api/java/lang/Comparable.html, it looks like you may also have to handle otherUser being null by throwing a  NullPointerException.